tests/e2e/specs/loans-spec.js

```diff
@@ -0,0 +1,71 @@
+import LoansPage from '../pages/loans-page';
+
+const loansPage = new LoansPage();
+
+describe('Loans', () => {
+	before(() => {});
+	context('Page', () => {
+		it('should be accessible from sidenav', () => {
+			cy.visit('/');
+			cy.findByTestId('sidenav-/loans').click();
+			cy.findByTestId('loans-form', { timeout: 30 * 1e3 }).should('be.visible');
+		});
+		it.only('should create & manage loans', () => {
+			loansPage.visit();
+			loansPage.getFormButton().contains('connect wallet').click();
+			cy.findByAltText('MetaMask').click();
+			loansPage.acceptMetamaskAccessRequest();
+			loansPage.getFormButton().contains('BORROW sUSD');
+
+			// switch debt to sETH
+			loansPage.getLeftInput().within(() => {
+				cy.findByTestId('select').click();
+				cy.findAllByText('sETH').click();
+			});
+			loansPage.getFormButton().contains('BORROW sETH');
+			loansPage.getLeftInput().contains('sETH');
+
+			// switch debt to sBTC
+			// should update default collateral to renBTC
+			loansPage.getLeftInput().within(() => {
+				cy.findByTestId('select').click();
+				cy.findAllByText('sBTC').click();
+			});
+			loansPage.getFormButton().contains('BORROW sBTC');
+			loansPage.getLeftInput().contains('sBTC');
+			loansPage.getRightInput().contains('renBTC');
+
+			// switch debt to sUSD
```

Comment:
it should be a separate test defined with `it('something')`

Reply:
Did split the test into smaller cases.